### DISCUSSION TITLE
not transfer video between diferent computers

### DISCUSSION THREAD

**Discussion by mlopezpalma** [bluenviron/mediamtx#1200] [Questions]
II am using the  next application   to send video  between diferent computers  linux   ufw , iptables  inactives  in both ) 

import cv2
from vidgear.gears import CamGear
from vidgear.gears import WriteGear

# open any valid video stream(for e.g `foo1.mp4` file)
stream = CamGear(source='pose.mp4').start()
#output_params = {"-vcodec": "libx264", "-profile:v": "main","-pix_fmt": "yuv420p", "-preset:v": "veryfast", "-g": 60, "-keyint_min": 60,
#                "-sc_threshold": 0, "-b:v": "2500k", "-maxrate": "2500k", "-bufsize": "2500k", "-f": "rtsp"}
output_params = {"-f": "rtsp", "-rtsp_transport":"udp"}
writer = WriteGear(output_filename='rtsp://0.0.0.0:8554/live/test', logging=True, **output_params)

# loop over
while True:
    # read frames from stream
    frame = stream.read()
    # check for frame if Nonetype
    if frame is None:
        break

    writer.write(frame)
    cv2.imshow("Output Frame", frame)

    key = cv2.waitKey(100) & 0xFF
    if key == ord("q"):
        break

cv2.destroyAllWindows()
stream.stop()
writer.close()

and for play 

ffplay rtsp://192.186.1.85:6554/live/test

. when is working in the same machine  all addres  to  127.0.0.1   or localhost     all works  ok .

But  in diferent machines   the  server shows the conection   ,  in both sender and receiver  ; the sender  is sending packets  ; but the receiver  is not receiving  any one  .

Some one  can help me please

**Comment by aler9** [comment]
Hello, use the TCP transport protocol and disable the UDP transport protocol by using this server configuration:

```yml
protocols: [tcp]
```

[a comment by an automated account is not shown]
